### ISSUE TITLE
Compilation fails with GCC6

### ISSUE THREAD

**Issue by agherzan** [upx/upx#19]
## What's the problem (or question)?
Compilation fails with GCC6.
```
| p_vmlinx.cpp: In static member function 'static int PackVmlinuxBase::compare_Phdr(const void, const void)':
| p_vmlinx.cpp:100:5: error: this 'if' clause does not guard... [-Werror=misleading-indentation]
| if (a->p_paddr > b->p_paddr) return 1;
| ^~
| p_vmlinx.cpp:101:34: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'if'
| return 0;
| ^~~~~~
```

## What should have happened?
upx should compile with GCC6.

## Do you have an idea for a solution?
werror can be removed
or
fix using the porting instructions available here: https://gcc.gnu.org/gcc-6/porting_to.html
or
compile with -Wmisleading-indentation

## How can we reproduce the issue?
Can be reproduced in a yocto build using https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-core/upx/upx_3.91.bb

## Help us understand your issue by providing context.
We are currently using upx to compress big binaries in resinOS.

## Please tell us details about your environment.
* UPX version used (`upx --version`): 3.19
* Operating System and version: linux
* CPU architecture: cross compiled for arm / x86-64 etc.

**Comment by markus-oberhumer** [comment]
This is already fixed on the devel branch, so closing.

**Comment by floion** [comment]
@markus-oberhumer can you point to the specific commit which fixed this please?

**Comment by floion** [comment]
Also, do you plan to tag/release a new version? I see the last one being done over 3 years ago.

**Comment by agherzan** [comment]
@markus-oberhumer Are you sure this was fixed in devel? I see a commit from 2006 at that specific line: https://github.com/upx/upx/blame/devel/src/p_vmlinx.cpp#L100

**Comment by markus-oberhumer** [comment]
See https://travis-ci.org/upx/upx/builds/171020082

**Comment by agherzan** [comment]
I confirm this is fixed in 44248f19b5291d6adc41be06ced081f8bd90f085 . It would be nice to have better commit log so we can trace things easier. As well, @markus-oberhumer about @floion 's question, do you plan to tag any new releases?
